includes/adapter/RecipientFromWCOrder.php

```diff
@@ -105,6 +105,10 @@ private function getAddressFromOrder(WC_Order $order, string $type): array
             $streetParts['number_suffix'] = $numberSuffix ?? $streetParts['number_suffix'] ?? null;
         }
 
+        if (is_int($streetParts['number_suffix'])) {
+            $streetParts['number_suffix'] = sprintf(' -%d', abs($streetParts['number_suffix']));
```

Comment:
...ik was aan het schrijven waarom ik dit raar en eng vond, maar ik snap nu dat dit een "hack" is om het streepje er vanaf te halen... clever, maar niet vanzelfsprekend, graag een comment erboven plaatsen. Als je de code leesbaar wilt maken moet je het helemaal uitschrijven dus in dit geval is een comment ook goed